### PR TITLE
Add benchmarker object

### PR DESCRIPTION
**Pull request by longjiaoz**
- add benchmarker object and the test
- add task_ids.json
- ass .gitignore and requirement.txt

### CODE REVIEW

File: benchmarkers/benchmarker.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    def get_search_space(self, seed=1):
+        self.config_space = self.benchmarker.get_configuration_space(seed)
+        config_space_dict = dict(self.config_space)
+        search_space_out = {}
+        for hp_name, space in config_space_dict.items():
+            # hp_dict = {}
+            # hp_dict["hyperparameter_name"] = hp_name
+            # hp_dict["range"] = self.parse_config_space(str(space))
+            # search_space_out.append(hp_dict)
+            search_space_out[hp_name] = self.parse_config_space(str(space))
```
Comment (longjiaoz) [inline benchmarkers/benchmarker.py]: change the range of search space from string to tuple

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+openai>=1.19.0
+requests>=2.31.0
+pillow>=10.3.0
+transformers>=4.39.3
+seaborn>=0.13.2
+loguru>=0.7.2
+xgboost>=2.0.3
+scikit-learn==1.1.3
+openml==0.13.1
```
Comment (longjiaoz) [inline requirements.txt]: add python-dotenv==1.0.1

File: benchmarkers/benchmarker.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            self.model_name = "XGBoost"
+            self.benchmarker = XGBoostBenchmark(task_id=task_id)
+            self.fidelity = {"n_estimators": 2000, "subsample": 1}
+        elif model_name == "svm":
+            self.model_name = "SupportVectorMachine"
+            self.benchmarker = SVMBenchmark(task_id=task_id)
+            self.fidelity = {"subsample": 1}
+        else:
+            raise Exception("Need a model name: xgb or svm")
+
```
Comment (roshansridhar) [inline benchmarkers/benchmarker.py:23]: we dont need lines 24 - 48
we just need one line `return dict((i.name, (i.lower, i.upper)) for i in self.benchmarker.get_configuration_space(seed).get_hyperparameters())`

**Review by roshansridhar — APPROVED** [review]
lgtm

**Pull request merged**